src/rpc/misc.cpp

```diff
@@ -597,6 +599,62 @@ static UniValue echo(const JSONRPCRequest& request)
     return request.params;
 }
 
+static UniValue SummaryToJSON(const IndexSummary&& summary, std::string index_name)
+{
+    UniValue ret_summary(UniValue::VOBJ);
+    if (index_name != "" && index_name != summary.name) {
+        return ret_summary;
+    }
+
+    UniValue entry(UniValue::VOBJ);
+    entry.pushKV("synced", summary.synced);
+    entry.pushKV("best_block_height", summary.best_block_height);
+    ret_summary.pushKV(summary.name, entry);
+    return ret_summary;
+}
+
+static RPCHelpMan getindexinfo()
+{
+    return RPCHelpMan{"getindexinfo",
+                "\nReturns the status of one or all available indices currently running in the node.\n",
+                {
+                    {"index_name", RPCArg::Type::STR, /* default */ "", "Filter results for an index with a specific name."},
```

Comment:
Would (something like) this be better?
```suggestion
                    {"index_name", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "Filter results for an index with a specific name."},
```

It would change the help from
```
Arguments:
1. index_name    (string, optional, default=) Filter results for an index with a specific name.
```
to
```
Arguments:
1. index_name    (string, optional) Filter results for an index with a specific name.
```

Reply:
(I should have given feedback about the "default=" earlier, but somehow kept forgetting to add it to the review)